packages/gatsby-transformer-react-docgen/src/on-node-create.js

```diff
@@ -10,6 +10,19 @@ const digest = str =>
 const propsId = (parentId, name) => `${parentId}--ComponentProp-${name}`
 const descId = parentId => `${parentId}--ComponentDescription`
 
+function canParse(node) {
+  return (
+    node &&
+    (node.internal.mediaType === `application/javascript` ||
+      // Typescript doesn't really have a mime type and .ts files are a media file :/
+      node.internal.mediaType === `application/typescript` ||
```

Comment:
Is there a reason we need _all_ of these checks? I would've thought there's one condition we could reliably use, i.e.

```js
return (
  node.internal.mediaType === `application/javascript` ||
  node.internal.mediaType === `application/typescript`
)
```